src/System.Linq.Parallel/tests/Helpers/Comparers.cs

```diff
@@ -122,4 +121,15 @@ public int CompareTo(DelegatedComparable<T> other)
             return _comparer.Compare(Value, other.Value);
         }
     }
+
+    internal struct NotComparable
+    {
+
+        public int Value { get; }
```

Comment:
get-only auto props were added in C# 6.  We're not building corefx yet using C# 6, so this fails to compile.